### ISSUE TITLE
OTA upgrade on IDO devices and other

### ISSUE THREAD

**Issue by sim1camporeale** [micooke/arduino-nRF5-smartwatches#13]
Do you think is it possibile to perform an OTA update and overwrite original firmware with my own firmware?

**Comment by micooke** [comment]
Yes, but not with this project. I haven't looked at it I'm sorry, but i know that this egg has been cracked by other people

**Comment by sim1camporeale** [comment]
Thanks, I ordered two device from another supplier that is 
V07s
V11 
from
**Wo-smart Technologies (Shenzhen) Co.,LTD**

I keep you updated as soon as they arrive.

**Comment by micooke** [comment]
No problem.
I don't have a lot of time to spend on this these days, but if you find a method for ota i am happy to try and integrate it.
I have also been meaning to upload a basic watch gui i designed a long time ago.. Been slack!
I am happy to accept PRs for pinouts or PRs in general. Cheers
